test/i-frame.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { browser } from 'protractor';
+import { IFramePage } from '../src/page';
+
+describe('Given a SDET learning protractor', () => {
+  const iFrame: IFramePage = new IFramePage();
+  describe('and then open Iframe page', () => {
+    beforeEach(async () => {
+      await browser.waitForAngularEnabled(false);
+      await browser.driver.get('https://demoqa.com/frames');
+    });
+
+    it('and then should have as title frames', async () => {
+      expect(await iFrame.getTittlePage()).toBe('Frames');
+    });
+    describe('and then set frame height', () => {
```

Comment:
```suggestion
    describe('And set frame height', () => {
```
This is an action and not a validation, so it should be a When instead of a Then. But to avoid to repeat the "When" (It is already in the previous action), just put the "And". There is no need to put And + When, or And + Then. (https://cucumber.io/docs/gherkin/reference/)